converters/google/convert.go

```diff
@@ -223,12 +267,51 @@ func (c *Converter) augmentAsset(tfData converter.TerraformResourceData, cfg *co
 		}
 	}
 
+	var orgPolicy []*OrgPolicy
+	if cai.OrgPolicy != nil {
+		for _, o := range cai.OrgPolicy {
+			var listPolicy *ListPolicy
+			var booleanPolicy *BooleanPolicy
+			var restoreDefault *RestoreDefault
+			if o.ListPolicy != nil {
+				listPolicy = &ListPolicy{
+					AllowedValues:     o.ListPolicy.AllowedValues,
+					AllValues:         ListPolicyAllValues(o.ListPolicy.AllValues),
+					DeniedValues:      o.ListPolicy.DeniedValues,
+					SuggestedValue:    o.ListPolicy.SuggestedValue,
+					InheritFromParent: o.ListPolicy.InheritFromParent,
+				}
+			}
+			if o.BooleanPolicy != nil {
+				booleanPolicy = &BooleanPolicy{
+					Enforced: o.BooleanPolicy.Enforced,
+				}
+			}
+			if o.RestoreDefault != nil {
+				restoreDefault = &RestoreDefault{}
+			}
+			currentTime := time.Now()
+			currentTime = currentTime.Round(time.Minute)
```

Comment:
Should add a comment about why we're rounding.

Reply:
Added a comment

Reply:
I don't see a comment?

Reply:
Actually I added comment in test file, added comment here as well